build.sh

```diff
@@ -2,28 +2,39 @@
 
 TAG="$(< VERSION)-$(git rev-parse --short HEAD)"
 
-echo "Getting updated images"
-docker-compose pull conjur_4 conjur_5
+echo "Getting updated images (this may take a few minutes)..."
+docker-compose pull -q conjur_4 conjur_5
 
 echo "Building Buildpack Health Check executable"
 rm -rf bin/buildpack-health-check
 docker-compose -f buildpack-health-check/docker-compose.yml build
 docker-compose -f buildpack-health-check/docker-compose.yml \
   run --rm buildpack-health-check-builder
 
-echo "Building conjur-service-broker Docker image"
-docker-compose build conjur-service-broker
-docker-compose build tests
+echo "Building conjur-service-broker image"
+docker build -t "conjur-service-broker:$TAG" \
+  -t "conjur-service-broker:latest" \
+  .
 
-echo "Tagging conjur-service-broker:$TAG"
-docker tag conjur-service-broker "conjur-service-broker:$TAG"
+echo "Building conjur-service-broker-test image"
+docker build -t "conjur-service-broker-test:latest" \
+  -f Dockerfile.test \
+  .
 
 echo "Running deployment build to install dependencies locally"
 echo "Creating project ZIP file"
-docker-compose run \
-  --rm \
+docker run --rm \
+  -v "$(pwd):/output" \
   conjur-service-broker \
-  bash -c "
-    bundle pack --all
-    zip -r cyberark-conjur-service-broker_$(cat VERSION).zip ./*
+  bash -ec "
+    echo 'Bundling dependencies...'
+    bundle package --all --no-install
```

Comment:
what does the additional `--no-install` do?

Reply:
This grabs all gems and caches them without installation. Installation is not needed since we already do it earlier and we also don't include the installed gems in the zip.